src/mockatoo/macro/VerifyMacro.hx

```diff
@@ -60,7 +60,7 @@ class VerifyMacro
 		var eIsNotNull:Expr;
 		var eIsAMock:Expr;
 
-		if (haxe.macro.Compiler.getDefine("cpp") != null)
+		if (Context.getDefines().get("cpp") != null)
```

Comment:
Better use `if (Context.defined("cpp"))`, as that doesn't need to construct the whole map ;)